Content.Server/Chat/Commands/NoticeCommand.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Server.Chat.Managers;
+using Content.Shared.Administration;
+using Content.Shared.Chat;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Enums;
+
+namespace Content.Server.Chat.Commands
+{
+    [AnyCommand]
+    internal sealed class NoticeCommand : IConsoleCommand
+    {
+        public string Command => "notice";
+        public string Description => "Send a chat message to self.";
+        public string Help => "notice <text>";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            if (shell.Player is not IPlayerSession player)
+            {
+                shell.WriteError("This command cannot be run from the server.");
+                return;
+            }
+
+            if (player.Status != SessionStatus.InGame)
+                return;
+
+            if (player.AttachedEntity is not { } playerEntity)
+            {
+                shell.WriteError("You don't have an entity!");
+                return;
+            }
+
+            if (args.Length < 1)
+                return;
+
+            var message = string.Join(" ", args).Trim();
+            if (string.IsNullOrEmpty(message))
+                return;
+
+            IoCManager.Resolve<IChatManager>().ChatMessageToOne(ChatChannel.Visual, message, message, EntityUid.Invalid, false, player.ConnectedClient, recordReplay: false);
+        }
+    }
+}
```

Comment:
Please update to code standards:
1. localisation
2. file-scoped namespace
3. public
4. Use dependencies not resolve.

Reply:
Make it a public class not internal please.